packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -140,13 +175,21 @@ Resources:
               AWS:
                 Fn::Sub: "${FilePublishingRole.Arn}"
             Resource: "*"
-    Condition: CreateNewKey
+    Condition: CreateCustomerMasterKey
   FileAssetsBucketEncryptionKeyAlias:
     Condition: CreateNewKey
     Type: AWS::KMS::Alias
     Properties:
       AliasName:
-        Fn::Sub: "alias/cdk-${Qualifier}-assets-key"
+        Fn::Sub: "alias/cdk-${Qualifier}-bucket-assets-key"
```

Comment:
Is this a breaking change? From the CFN docs

> Adding, deleting, or updating an alias can allow or deny permission to the KMS key.

I also am not sure it's necessary.

Reply:
True! Let's revert this then and keep the old name.